ingredients.py

```diff
@@ -31,32 +34,36 @@ def load_mnist(digits, fraction=1.0, rotation=0):
                                         transforms.Lambda(lambda x: x.view(28**2))
                                         ])
 
-    if rotation is 15:
+    elif rotation is 15:
```

Comment:
All of rotation stuff is not very good code, plus you brought in the dependency of torchsample which is unneccesary.